---
layout: page
title: Station A.  Experimental Design
---
## Introduction to Genome Station A. Experimental Design

Naturally, there are no tools to help a researcher determine experimental intention, but this is the most critical station on the train ride to a reference genome. Here is a checklist of important things to consider:

**1. How will you use the reference genome reagent?**

  Is there a specific hypothesis in mind, or is this an expedition to form new ones?
  
  Is the genome part of a cluster of closely related species, or just a favourite one?

**2. How will you identify and engage the user community?**

  Key advisors that can help along the way?
  Are there any existing community resources, such as ESTs, BAC libraries, frozen specimens or live cultures?
  Are there any closely related species with sequenced genomes?
  Are there any closely related species with sequenced AND well-annotated genomes?

**3. What do you know about the genome?**

  Any prior knowledge on the genomic architecture of the species (e.g. sex chromosomes, ploidy, genome size, number of chromosomes, genome complexity)?
  Which exact individual will be the reference genome source (e.g. sex, heterozygosity levels, neat phenotype).

**4. How will you acquire high quality DNA and RNA material?**

  Have established an optimum extraction protocols and best practices. Which specimens to use and how many?
  Which tissues for expression?
  Access to haploid tissues?
  Is it required (and possible) to establish an inbred line to reduce heterozygosity?

**5. How much and what kind of sequence data will you generate?**

  Based upon your genome size and complexity, how much raw sequece data is needed and of what kind?
  What genome sequencing methods are available to you? Are they experimental or reliable and established?
  What are the library preparation options?
  Check whether DNA/RNA yields are compatible proposed library preparation and sequencing platforms
  Do you have enough money to sequence genomic DNA and RNA libraries?
  What can be done commercially and what in-house?
  Do you have bioinformatics support and enough computational power?
  
**6. What are the personal aims that need to be achieved (e.g. a publication, specific downstream experiments etc)?**
