At this time, DNA sequencing technology cannot determine the complete nucleotide sequence of all chromosomal DNA strands from a sample of target organism DNA.    Chromosomes are simply too large, so one must first physically break the DNA strands into pieces, sequence, and reconstruct the reference chromosome sequences.

Genome assemblers are complex software workflows that require high-performance computing resources, significant operator skill, and real experience in optimizing the genome reconstruction process, which can be very different depending upon the genome complexity, heterozygosity and ploidy of a given species.    

At this train station, we discuss the process of putting end sequences from chromosome molecular fragments back together again into a reference genome assembly.  It should be noted that no reference genome is perfect.  It will have gaps and misassemblies.  These experimental errors can sometimes be detected & corrected (i.e. finished) using bioinformatic and molecular biology approaches.

With this caveat that no genome assembly is perfect, it is up to the genome assembly team to decide what the acceptable level of experimental error can be allowed to address the requirements of the experimental design.  Thus, we will also discuss objective tools to ensure the genome assembly is “good enough” and not just assume that it is of sufficient quality because it’s a “lot of data”. 

