The aims of structural feature annotation may appear rather straight forward – to identify DNA sequences along the reference chromosome sequences which encode biological function – yet producing a reliable set of structural annotations involves a complex combination of ab initio gene prediction, assessment of the available biological evidence to drive the gene prediction process, and the synthesis of these results to produce gene annotations. Faced with the plethora of tools designed to perform such tasks, deciding how to go about it can be quite daunting.

Before choosing an annotation strategy it is important to consider all available resources, for example, whether transcriptomic data (e.g. ESTs, full-length mRNA, RNA-Seq) or protein data are available to feed into feature annotation tools. Or, whether trusted gene models from well-annotated genomes of closely related species are available for comparison. 
